utils/api.ts

```diff
@@ -1,6 +1,7 @@
-import { ApiPromise, WsProvider } from '@polkadot/api'
+import { ApiPromise } from '@polkadot/api'
 import { testLog } from './Logger';
 import { getEnvironmentRequiredVars } from './utils'
+import {Mangata} from 'mangata-sdk'
```

Comment:
Did you run your linter ? Those brackets should be `{ Mangata }`

Reply:
We have linter but not really strict.
Should we increment the error policy? I used to have info- as error, but sometimes is overkill :o: 
What do you think @eddyjlsh  ?

Reply:
great ;)